examples/complete/main.tf

```diff
@@ -1,78 +1,59 @@
-
-terraform {
-  required_providers {
-    github = {
-      source = "integrations/github"
-    }
-  }
-}
-
-data "github_team" "admin" {
-  slug = "AdminTestTeam"
-}
-
-data "github_team" "maintain" {
-  slug = "MaintainTestTeam"
-}
-
-data "github_team" "contributer" {
-  slug = "ContributerTestTeam"
-}
-
-locals {
-  admin      = data.github_team.admin.id
-  maintain   = data.github_team.maintain.id
-  conributer = data.github_team.contributer.id
-}
-
-
-resource "random_string" "repository" {
-  length  = 5
-  special = false
-  upper   = false
-  number  = false
-}
-
-# ###############################################################################
-# The examples below create an organization repository and team
-# To deploy this example export both the GITHUB_TOKEN and GITHUB_OWNER variables
-# ###############################################################################
-
-module "random_repository" {
-  source             = "boldlink/repository/github"
-  version            = "1.0.0"
-  name               = "new-repo"
-  description        = "Terraform random repository example"
-  license_template   = "apache-2.0"
-  allow_squash_merge = true
-  branch             = ["dev", "pre", "prd"]
-  default_branch     = "develop"
-  gitignore_template = "Terraform"
-  visibility         = "public"
-  homepage_url       = "https://boldlink.io"
-  teams = {
-    admin    = local.admin
-    maintain = local.maintain
-  }
-  required_pull_request_reviews = {
-    dismiss_stale_reviews           = true
-    require_code_owner_reviews      = true
-    required_approving_review_count = 1
-    dismissal_teams                 = []
-    dismissal_users                 = []
-  }
-  restrictions = {
-    users = []
-    teams = [
-      local.admin,
-      local.maintain
-    ]
-    apps = []
-  }
-}
-
-output "outputs" {
-  value = [
-    module.random_repository,
-  ]
-}
\ No newline at end of file
+
+# Sample organization teams
+resource "github_team" "admin" {
+  name        = "admin-team"
+  description = "Sample admin team"
+  privacy     = "closed"
+}
+
+resource "github_team" "maintain" {
+  name        = "maintain-team"
+  description = "Sample maintain team"
+  privacy     = "closed"
+}
+
+locals {
+  admin    = github_team.admin.id
+  maintain = github_team.maintain.id
+}
+
+# ###############################################################################
+# The examples below create an organization repository and team repository
+# To deploy this example export both the GITHUB_TOKEN and GITHUB_OWNER variables
+# ###############################################################################
+
+module "complete" {
+  source             = "./../../"
+  name               = "example-complete"

```

Comment:
Although this creates the repo, what about making the name unique by using random or random pet?